src/de/uni_hamburg/informatik/swt/se2/kino/werkzeuge/kasse/KassenWerkzeug.java

```diff
@@ -119,4 +126,19 @@ private Vorstellung getAusgewaehlteVorstellung()
     {
         return _vorstellungAuswaehlWerkzeug.getAusgewaehlteVorstellung();
     }
+
+	@Override
+	public void reagiereAufAenderung(Observable obj) {
+		
```

Comment:
`assert obj != null : "Vorbedingung verletzt: obj != null"`

Reply:
finde ich nicht notwendig, weil es ja sonst einfach am if block vorbeirauscht ohne einen Fehler zu werfen. Außer wir wollen in dem Fall das Programm anhalten...